### PR TITLE
ircd-hybrid 8.2.12

### PR DESCRIPTION
**Pull request by bartleusink**
Hi!

This PR upgrades ircd-hybrid from 8.2.11 to 8.2.12

Changelog: https://raw.githubusercontent.com/ircd-hybrid/ircd-hybrid/master/NEWS

### CODE REVIEW

File: Library/Formula/ircd-hybrid.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class IrcdHybrid < Formula
   desc "High-performance secure IRC server"
   homepage "http://www.ircd-hybrid.org/"
-  url "https://downloads.sourceforge.net/project/ircd-hybrid/ircd-hybrid/ircd-hybrid-8.2.11/ircd-hybrid-8.2.11.tgz"
-  sha256 "eeb916ca22b0ee2c7b36e00249ca41001a82587597084c102fe11ad8e85cfe78"
+  url "https://github.com/ircd-hybrid/ircd-hybrid/archive/8.2.12.zip"
```
Comment (bfontaine) [inline Library/Formula/ircd-hybrid.rb]: We could use `.tar.gz` instead of `.zip` to get a smaller file (~1Mo vs. ~1.3Mo).

Comment (bartleusink) [inline Library/Formula/ircd-hybrid.rb]: You're absolutely right, I've changed the url to point to the `tar.gz` now

File: Library/Formula/ircd-hybrid.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 class IrcdHybrid < Formula
   desc "High-performance secure IRC server"
   homepage "http://www.ircd-hybrid.org/"
-  url "https://downloads.sourceforge.net/project/ircd-hybrid/ircd-hybrid/ircd-hybrid-8.2.11/ircd-hybrid-8.2.11.tgz"
-  sha256 "eeb916ca22b0ee2c7b36e00249ca41001a82587597084c102fe11ad8e85cfe78"
+  url "https://github.com/ircd-hybrid/ircd-hybrid/archive/8.2.12.tar.gz"
```
Comment (DomT4) [inline Library/Formula/ircd-hybrid.rb]: Is this an official release mirror on their side, rather than just automatically generated tarballs?

Comment (bartleusink) [inline Library/Formula/ircd-hybrid.rb]: You're right, I've changed the url back to sourceforge.

Comment (DomT4) [inline Library/Formula/ircd-hybrid.rb]: Thanks! We've had upstreams grumble at us about premature switching before so we're trying to stay as consistent as possible with what upstream display as the download link, etc.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/d23cf269a5d50c8ee3f1e6e0eb8cd8d870c9681a. Thank you for another contribution to Homebrew @bartleusink! :smiley_cat:

**Pull request closed without merging**